assets/components/molecules/nav/nav.scss

```diff
@@ -0,0 +1,371 @@
+@charset 'utf-8';
+
+//
+// Inner nav styles
+//
+.nav {
+  background: white;
+  z-index: 100;
+  transition: left 0.3s;
+  border-right: 1px solid gray('100');
+
+  .nav-wrapper {
+    width: 100%;
+    height: 100%;
+    overflow: scroll;
+    overflow: -moz-scrollbars-none;
+    -ms-overflow-style: none;
+
+    &::-webkit-scrollbar {
+      display: none;
+    }
+  }
+
+  .nav-container {
+    display: block;
+    position: relative;
+    width: 100%;
+    margin-left: -100%;
+    padding-left: 100%;
+  }
+
+  .current_page_ancestor > ul {
+    display: block;
+    left: 0;
+  }
+
+  .current_page_parent > ul {
+    display: block;
+    left: 100%;
+  }
+
+  ul {
+    $nav-spacer: 4px;
+
+    display: none;
+    position: absolute;
+    top: 0;
+    left: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 0 2rem 0 3rem;
```

Comment:
We have a lot of mixed `$spacer` and `rem` values, maybe find a way to unify that?

Reply:
rem is gone! 
used $spacer or $font-size-base instead